zeppelin/legacy_converter.py

```diff
@@ -0,0 +1,88 @@
+import re
+import os
+import cairosvg
+from .converter import ZeppelinConverter
+
+
+class LegacyConverter(ZeppelinConverter):
+    """LegacyConverter converts Zeppelin version 0.6.2 notebooks to Markdown."""
+
+    MD_EXT = '.md'
+
+    def build_image(self, msg):
+        """Convert svg encoding to png.
+
+        Uses CairoSVG to convert svg image to a png and outputs
+        the images to the specified directory.
+        """
+        result = re.search('xml version', msg)
+        if result is None:
+            return
+
+        self.index += 1
+        images_path = 'images'
+
+        if self.directory:
+            images_path = os.path.join(self.directory, images_path)
+
+        if not os.path.isdir(images_path):
+            os.makedirs(images_path)
+
+        with open('{0}/output_{1}.png'.format(images_path, self.index), 'wb') as fh:
+            cairosvg.svg2png(bytestring=msg.encode('utf-8'), write_to=fh)
+
+        self.out.append(
+            '\n![png]({0}/output_{1}.png)\n'.format(images_path, self.index))
+
+    def build_text(self, msg):
```

Comment:
In LegacyConverter, it's `self.out.append(msg)`, whereas in NewConverter, its `self.out.append(msg['data'])`. A bit different; not sure exactly how to proceed with this, since the JSON formats are a bit different.